### PR TITLE
Fix/upgrade android sdk

### PR DESCRIPTION
**Pull request by adyaverloop**

### CODE REVIEW

File: android/app/google-services.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (raghav39) [inline android/app/google-services.json]: We should never add google-services.json to the git repo

File: android/src/main/kotlin/io/verloop/verloop_flutter_sdk/ButtonClickHandler.kt [hunk 2]
```diff
@@ -1,31 +1,31 @@
-package io.verloop.verloop_flutter_sdk
```
Comment (raghav39) [inline android/src/main/kotlin/io/verloop/verloop_flutter_sdk/ButtonClickHandler.kt:1]: What changed here?

File: android/local.properties [hunk 3]
```diff
@@ -0,0 +1,3 @@
+sdk.dir=C:\\Users\\ADYA H S\\AppData\\Local\\Android\\sdk
```
Comment (raghav39) [inline android/local.properties]: local properties should not be added to this

File: android/app/build.gradle [hunk 4]
```diff
@@ -0,0 +1,77 @@
+def localProperties = new Properties()
```
Comment (raghav39) [inline android/app/build.gradle]: Shouldn't these be in example app instead of android wrapper?

**Review by raghav39 — CHANGES_REQUESTED** [review]

File: android/app/build.gradle (hunk 4 above)
Comment (adyaverloop) [reply android/app/build.gradle]: these are in the example /android/app/build.gradle

File: android/app/google-services.json (hunk 1 above)
Comment (adyaverloop) [reply android/app/google-services.json]: will remove it

File: android/src/main/kotlin/io/verloop/verloop_flutter_sdk/ButtonClickHandler.kt (hunk 2 above)
Comment (adyaverloop) [reply android/src/main/kotlin/io/verloop/verloop_flutter_sdk/ButtonClickHandler.kt:1]: nothing as such

File: android/build.gradle [hunk 5]
```diff
@@ -1,52 +1,52 @@
...
+    }
+
+    defaultConfig {
+        minSdkVersion 19
+    }
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.github.verloop:android-sdk:1.1.1-rc.2'
```
Comment (raghav39) [inline android/build.gradle]: Need to upgrade this android SDK version

**Review by raghav39 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**